src/content/docs/extensions/algos/wcc.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Weakly Connected Components
+---
+
+A weakly connected component (WCC) in a directed graph is a maximal subgraph where any two vertices
+are connected to each other by some path, regardless of edge direction.
+
+Kuzu implements a parallelized version of the WCC algorithm based on [Ligra](https://jshun.csail.mit.edu/ligra.shtml).
```

Comment:
Do we need to mention this?

Reply:
Who is this a question for @ray6080?

Reply:
> Who is this a question for @ray6080?

I meant for @andyfengHKU

Reply:
I think it's fine. See for example https://neo4j.com/docs/graph-data-science/current/algorithms/wcc/